android/app/src/main/res/values/strings.xml

```diff
@@ -50,7 +50,7 @@
     <string name="record_vaccine">Record Vaccine</string>
     <string name="no_vaccine_received">No vaccine received</string>
     <string name="view_registration_details">View Registration Details</string>
-    <string name="covax_client">COVAX clients</string>
+    <string name="client">COVAX clients</string>
```

Comment:
@syedowaisali90 Do we mean to change the Text to `Clients` as well or is it not needed for now?

Reply:
I thought the issue was related to the naming convention because we will use the same codebase in different flavors like `CHW/ANC` that's why we have to follow a generic pattern. @f-odhiambo can confirm

Reply:
yea I think we can leave it as now and clean this up when we move to a configuration based label here